bin/SConsDoc.py

```diff
@@ -817,11 +865,10 @@ def importfile(path):
 else:  # PY3 version, from newer pydoc
     def importfile(path):
         """Import a Python source file or compiled file given its path."""
-        import importlib
-        from pydoc import ErrorDuringImport
-        magic = importlib.util.MAGIC_NUMBER
+        from importlib import MAGIC_NUMBER
```

Comment:
should that be importlib.util and not importlib?

Reply:
yes